spec/rubocop/cop/performance/redundant_merge_spec.rb

```diff
@@ -61,6 +65,25 @@
       expect(cop.messages).to eq(
         ["Use `hash[:a] = 1` instead of `hash.#{method}(a: 1)`."])
     end
+
+    context 'with SingleOnly switch' do
+      let(:cop_config) do
+        { 'MaxKeyValuePairs' => 1 }
+      end
+
+      it "doesn't autocorrect multi-value hash merges" do
```

Comment:
This example is redundant. You only need to check whether the cop registers offences or not.